src/OutputWriters/netcdf_output_writer.jl

```diff
@@ -49,16 +49,59 @@ zdim(::Nothing) = tuple()
 netcdf_spatial_dimensions(::AbstractField{LX, LY, LZ}) where {LX, LY, LZ} =
     tuple(xdim(instantiate(LX))..., ydim(instantiate(LY))..., zdim(instantiate(LZ))...)
 
-const c = Center()
-const f = Face()
-
-native_dimensions_for_netcdf_output(grid, indices, TX, TY, TZ, Hx, Hy, Hz) =
-    Dict("xC" => nodes(grid, (c, c, c))[1],
-         "xF" => nodes(grid, (f, c, c))[1],
-         "yC" => nodes(grid, (c, c, c))[2],
-         "yF" => nodes(grid, (c, f, c))[2],
-         "zC" => nodes(grid, (c, c, c))[3],
-         "zF" => nodes(grid, (c, c, f))[3])
+function native_dimensions_for_netcdf_output(grid, indices, TX, TY, TZ, Hx, Hy, Hz)
```

Comment:
I wonder if, as @glwagner suggested, it wouldn't be better to have `ξnodes`, `ηnodes` and `rnodes` that return the native coordinates of the grid (as we have now `ξnode`, `ηnode`, and `rnode`). That would avoid needing to specialize this function

Reply:
Agreed and already done!